controllers/grid/navigationMenus/form/PKPNavigationMenuItemsForm.php

```diff
@@ -180,19 +180,44 @@ public function execute(...$functionParams)
             $navigationMenuItem = $navigationMenuItemDao->newDataObject();
             $navigationMenuItem->setTitle($this->getData('title'), null);
         } else {
+            // The NMI will have localized title in the navigation_menu_item_settings table
+            // only if the user is defining one explicitely.
+            // This is not always the case for the default NMIs that, by default, get their titles
+            // from the titleLocaleKey setting of the NMI.
+
+            // Get the title that have been explicitely set in the settings tatble
+            $localizedTitlesFromDB = $navigationMenuItem->getTitle(null) ?? [];
+
+            // Get the set of all displayed titles, no matter the source. 
+            Services::get('navigationMenu')
+                ->setAllNMILocalizedTitles($navigationMenuItem);
+            $originalDisplayTitles = $navigationMenuItem->getTitle(null);
```

Comment:
As this is basically the same as the variable `$localizedTitlesFromDB`, I think it can be removed.

Reply:
I just remembered this code is cursed, ignore the comment 🫠